mantidproject/mantid
# BayesQuasi skipping system and unit tests

Fixes #14476 

BayesQuasi and QLRun should now be properly covered by system and unit tests

**Files available from `\ExternalData\Testing\Data\UnitTest` folder**
# To Test
- Open Quasi (Interfaces > Indirect > Bayes > Quasi)
- Input=`irs26176_graphite002_red.nxs`
- Resolution=`irs26173_graphite002_res.nxs`
- Click `Run` - _default values in the interface will be fine for this test_
- Ensure that the algorithm runs to completion without errors
- Ensure that the system tests `ISISIndirectBayes`, unit test `BayesQuasi`and doctest `BayesQuasi-v1` all run on the build servers and pass
